### PR TITLE
Fix #2820: Add label for FAQSingleActivity

### LINKED ISSUE #2820: [A11y] Add label for FAQSingleActivity
## Task
Introduce label for `FAQSingleActivity` as `FAQs (Frequently Asked Questions)`.

Reference PR: https://github.com/oppia/oppia-android/pull/2750

When you create a PR for this, make sure you add test to check the functionality. Also, suggest working on this PR only if its your first or second PR to Oppia-Android project.

## General Explanation
When `Talkback` is on, whenever we click on any option which leads to new screen the reader will automatically read the `label` associated with that screen first. If in case label in not present then it will read app name i.e. `Oppia`.

### Example

https://user-images.githubusercontent.com/9396084/108906068-78b75080-7646-11eb-943e-212e29fc3e98.mp4

In this above video as soon as we double-tap `Battery`, new screen is opened. 
The reader first outputs `Battery` followed by `Navigate up, Button` and finally `Battery` again. In this the first `Battery` output is the `label` of this new screen which indicates the learner which screen is opened. The second `Battery` is because we selected the title in the screen and it's just reading that text.
So we need `labels` for various screens general which matches the title but if the title is dynamic then we need static label.

### PR DESCRIPTION
**Pull request by yashraj-01**
## Explanation

Fixes #2820 . Added label for FAQSingleActivity
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by yashraj-01** [comment]
@rt4914 @BenHenning PTAL

File: app/src/sharedTest/java/org/oppia/android/app/faq/FAQSingleActivityTest.kt [hunk 1]
```diff
@@ -104,6 +107,15 @@ class FAQSingleActivityTest {
     Intents.release()
   }
 
+  @Test
+  fun testFAQSingleActivity_hasCorrectActivityLabel() {
```
Comment (BenHenning) [inline app/src/sharedTest/java/org/oppia/android/app/faq/FAQSingleActivityTest.kt]:
```suggestion
  fun testFaqSingleActivity_hasCorrectActivityLabel() {
```

The activity name itself will need to get updated in a different PR, but we should follow the camel-casing rules defined here: https://developer.android.com/kotlin/style-guide#camel_case.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @yashraj-iitr! just had one comment.

File: app/src/sharedTest/java/org/oppia/android/app/faq/FAQSingleActivityTest.kt (hunk 1 above)
Comment (yashraj-01) [reply app/src/sharedTest/java/org/oppia/android/app/faq/FAQSingleActivityTest.kt]: Done. PTAL. Thanks.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -32,6 +32,7 @@
       android:label="@string/faq_activity_label" />
     <activity
       android:name=".app.help.faq.faqsingle.FAQSingleActivity"
+      android:label="@string/faq_activity_label"
```
Comment (rt4914) [inline app/src/main/AndroidManifest.xml]: Rename string to `faq_activity_title`

Comment (rt4914) [reply app/src/main/AndroidManifest.xml]: Also update it other places where its used.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@YashKr01 PTAL

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (yashraj-01) [reply app/src/main/AndroidManifest.xml]: Done. PTAL.

**Review by BenHenning — APPROVED** [review]
Thanks @yashraj-iitr! LGTM.

**Comment by yashraj-01** [comment]
@anandwana001 should I add screenshots of test results here as well?

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**